api/aladin/admin.py

```diff
@@ -0,0 +1,10 @@
+from django.contrib import admin
+from .models import Image
+
+
+class ImageAdmin(admin.ModelAdmin):
+    list_display = ['id', 'img_url', 'product']
+    list_display_links = ['id', 'img_url', 'product']
```

Comment:
ok vou retirar, estou atualizando a doc
https://github.com/linea-it/dri/wiki/Creating-a-new-app#new-model

vou atualizar também o branch de commit por posição, para manter isso padrão